data/db-schema/src/main/resources/net/corda/db/schema/rbac/migration/rest-rbac-creation-v1.0.xml

```diff
@@ -3,15 +3,15 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
 
-    <property name="schema.name" value="RPC_RBAC" global="false"/>
+    <property name="schema.name" value="REST_RBAC" global="false"/>
 
-    <changeSet author="vkolomeyko" id="rpc-perm-creation-v1.0">
+    <changeSet author="vkolomeyko" id="rest-perm-creation-v1.0">
 
         <sql>
-            CREATE SCHEMA IF NOT EXISTS RPC_RBAC;
+            CREATE SCHEMA IF NOT EXISTS REST_RBAC;
         </sql>
 
-        <createTable tableName="rpc_user" schemaName="${schema.name}">
+        <createTable tableName="rest_user" schemaName="${schema.name}">
```

Comment:
Should be either "user" or "rbac_user" IMO.

Same for the rest of the changes to "rest_" because the rbac stuff isn't only rest- worker.

IMO you could leave this change out, its not that important.